test/libsolidity/syntaxTests/nameAndTypeResolution/416_interface_function_bodies.sol

```diff
@@ -4,4 +4,4 @@ interface I {
 }
 // ----
 // TypeError: (18-45): Functions in interfaces cannot have an implementation.
-// Warning: (18-45): Functions in interfaces should be declared external.
+// TypeError: (18-45): Functions in interfaces must be declared external.
```

Comment:
I think in this test the function should be declared external - the test is only about the function having a body.

Reply:
yeah, you are right!